lua/lvim/core/treesitter.lua

```diff
@@ -16,7 +16,7 @@ M.config = function()
       disable = { "latex" },
     },
     context_commentstring = {
-      enable = false,
+      enable = true,
       config = { css = "// %s" },
```

Comment:
```suggestion
      config = {
            -- Languages that have a single comment style
            typescript = '// %s',
            css = '/* %s */',
            scss = '/* %s */',
            html = '<!-- %s -->',
            svelte = '<!-- %s -->',
            vue = '<!-- %s -->',
            json = ''
      },
```
do you think we should add these? although it has a different style for css, so I'll leave it to you since I don't use any of these languages.

Reply:
sure, looks good to me, if there are complaints one way or the other we can update this list